docs/design/configuration/architecture_file.md

```diff
@@ -100,3 +106,22 @@ nodes:
         subscription_topic_name: /pong
         publisher_topic_name: /ping
 ```
+
+## Callback identification
+
+`callback_name` is a name used by users to identify a callback.
+
+CARET reads trace data and loads callback information.
+Callback addresses can be obtained from trace data,
+but callback_name cannot be obtained because ROS does not have the ability to assign callback_names.
+
+After loading information, CARET binds the `callback_name` and callback address by following items.
```

Comment:
In order to tackle the issue, CARET associates a name with an address of callback with using combination of following data.

Reply:
```suggestion
In order to tackle the issue, CARET associates a name with an address of callback with using combination of following data.
```